fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/data/ConfigurationData.java

```diff
@@ -95,14 +95,20 @@ public CCell getCellConfig() {
     /**
      * Checks if server has been configured in network.json and
      * return it.
+     * Note: This returns a copy of the configuration in case multiple servers
+     * use the same configuration.
      *
      * @param serverGroupName name of the defined group in mapping
      * @return the configured {@code CNetworkProperties} if server region exists, else {@code null}
      */
-    public CNetworkProperties getServerRegion(String serverGroupName) {
+    public CNetworkProperties getServerRegionFromConfiguration(String serverGroupName) {
```

Comment:
@kschrab can you verify, that this is doing what I want it to do 😄

Reply:
You mean the `clone()` thing? I would guess not.. :( As far as I know, the default implementation of `Object.clone()` does copy the values of primitive members and references of other members to the new object only. That is, CUplink and CDownlink would be the same objects in the cloned CNetworkProperties.

What you would need is a shallow copy, which you would need to create by your own. You do not need to use the Clonable interface, but could just build the copy directly here.

Reply:
turned out to be unnecessary. Adding unit test though.